docs/sources/tempo/operations/troubleshooting/metrics-generator.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Troubleshoot Metrics Generator
+menuTitle: Metrics Generator
+description: Gain an understanding of how to debug metrics quality issues.
+weight: 500
+---
+
```

Comment:
Consider adding a first-leveling heading for the page. This heading should match the page title. 
```suggestion
# Troubleshoot metrics-generator

```